src/github.com/getlantern/proxiedsites/proxiedsites.go

```diff
@@ -1,269 +1,217 @@
-// package proxiedsites is a module used to manage the list of sites
-// being proxied by Lantern
-// when the list is modified using the Lantern UI, it propagates
-// to the default YAML and PAC file configurations
+// proxiedsites manages a list of proxied sites, including a default set of
+// sites (cloud) and user-applied customizations to that list. It provides an
+// implementation of the http.Handler interface that serves up a PAC file based
+// on the currently active proxied sites (cloud + additions - deletions).
 package proxiedsites
```

Comment:
BTW, I try to be good about logging stuff like this, so if you're ever looking to see where stuff is coming from, the log output is useful:

```
flashlight: Placing logs in /Users/ox.to.a.cart/Library/Logs/Lantern
Feb 20 17:56:27.919 - flashlight: ---- flashlight version development (now) ----
Feb 20 17:56:27.919 - flashlight.config: Placing configuration in /Users/ox.to.a.cart/Library/Application Support/Lantern
```